docs/api/RESTAPI.md

```diff
@@ -48,7 +48,7 @@ The `workflow_url` is either an absolute URL to a workflow file that is accessib
 
 The `workflow_params` JSON object specifies input parameters, such as input files.  The exact format of the JSON object depends on the conventions of the workflow language being used.  Input files should either be absolute URLs, or relative URLs corresponding to files uploaded using `workflow_attachment`.  The WES endpoint must understand and be able to access URLs supplied in the input.  This is implementation specific.
```

Comment:
If you look at the test failures attached to this PR, `checkPublish` has a failure referencing this file being out of date - I think it's autogenerated and shouldn't be edited directly. `project/GenerateRestApiDocs.scala` is the place the autogen task is defined, that should make it clear what needs to be changed.